core/src/tile/tile.h

```diff
@@ -51,7 +49,7 @@ class Tile {
     float getScale() const { return m_scale; }
 
     /* Returns the reciprocal of <getScale()> */
-    float getInverseScale() const { return m_inverseScale; }
+    float getInverseScale() const { return 1.0f / m_scale; }
```

Comment:
I'll remove this change as it does not really belong into this PR.

Thinking about the meaning of scale vs inverse scale it's not quite clear to me why tile width in projection-meters is considered to be the 'scale' of a tile - Is this an arbitrary decision or is there a good reasoning for it? 

If the distiction is arbitrary I would suggest to remove the m_scale member instead of m_inverseScale and rename inverseScale to scale. Since m_scale and tile.getScale() are not used anywhere, except for initialization of the tile's modelmatrix  :)